data/particle/property/particle_one.dat

```diff
@@ -0,0 +1 @@
+0.0000 0.0000 0.0000
```

Comment:
Is this a useful addition?

Reply:
Could you add a Readme.md file that gives a two line description of what this file does?